cli/internal/packager/deploy.go

```diff
@@ -258,3 +285,61 @@ func isValidFileExtension(filename string) bool {
 
 	return false
 }
+
+func injectManifestSecrets(path string) {
+	gitSecret := git.GetOrCreateZarfSecret()
+
+	zarfHtPassword, err := utils.GetHtpasswdString(config.ZarfGitUser, gitSecret)
+	if err != nil {
+		logrus.Debug(err)
+		logrus.Fatal("Unable to define `htpasswd` string for the Zarf user")
+	}
+	zarfDockerAuth := base64.StdEncoding.EncodeToString([]byte(fmt.Sprintf("%s:%s", config.ZarfGitUser, gitSecret)))
+
+	// Get a list of all the k3s manifest files
+	manifests := utils.RecursiveFileList(path)
+
+	// Iterate through all the manifests and replace any ZARF_SECRET, ZARF_HTPASSWD, or ZARF_DOCKERAUTH values
+	for _, manifest := range manifests {
+		logrus.WithField("path", manifest).Info("Processing manifest file")
+		utils.ReplaceText(manifest, "###ZARF_SECRET###", gitSecret)
+		utils.ReplaceText(manifest, "###ZARF_HTPASSWD###", zarfHtPassword)
+		utils.ReplaceText(manifest, "###ZARF_DOCKERAUTH###", zarfDockerAuth)
+	}
+}
+
+func runScript(script string) {
+	_, err := utils.ExecCommand(true, nil, "sh", "-c", script)
+	if err != nil {
+		logrus.Debug(err)
+		logrus.WithField("script", script).Fatal("Unable to run the component script")
+	}
+}
+
+func loopScript(script string) {
+	logContext := logrus.WithField("script", script)
+	logContext.Info("Waiting for ready script to complete successfully")
+
+	// 2 minutes per script (60 * 2 second waits)
+	tries := 60
+	for {
+		tries--
+		// If there are no more tries left, drop a warning and continue
+		if tries < 1 {
+			logContext.Warn("Ready script timed out after 2 minutes")
```

Comment:
I don't think this actually breaks out of the infinite for loop  after the number of tries has been depleted.

Reply:
it didnt' thanks, I fixed yesteray